canonical-websites/www.ubuntu.com
# Generate breadcrumbs from navigation.yaml

Use the new navigation.yaml to generate breadcrumbs smartly, so you don't need to specify them on every page.

**NB:** Make sure #1942 is merged before reviewing this one..

Demo: http://www.ubuntu.com-navigation-breadcrumbs.demo.haus/

Fixes #1944.

QA
--

Read the HACKING.md section about navigation, check it makes sense.

`./run` and browse the site. Check breadcrumbs behave as expected. Especially try:

- contact-us and thank-you pages. These are "hidden" from other navigations, and shouldn't display their breadcrumb siblings.
- "child" pages like `/cloud/managed-cloud`
- "child" pages with children, like `/cloud/openstack`
- "grandchild" pages like `/download/cloud/conjure-up`

- Try changing something in `navigation.yaml`, check the breadcrumbs respond as you would expect (you'll need to reload the server to see changes).
